circle.yml

```diff
@@ -0,0 +1,128 @@
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: circleci/node:8.9.1
+
+whitelist: &whitelist
+  paths:
+    .
+
+version: 2
+jobs:
+  checkout:
+    <<: *defaults
+
+    steps:
+      - checkout
+
+      # - run:
```

Comment:
I do not intend to use JFrog here (opensource). I do intend to use `npmjs.com`

I need to figure out how best to handle these credentials in Circle CI.